ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -500,6 +512,7 @@ func resourceIBMContainerVpcWorkerPoolRead(d *schema.ResourceData, meta interfac
 	if workerPool.WorkerVolumeEncryption != nil {
 		d.Set("kms_instance_id", workerPool.WorkerVolumeEncryption.KmsInstanceID)
 		d.Set("crk", workerPool.WorkerVolumeEncryption.WorkerVolumeCRKID)
+		d.Set("kms_account_id", workerPool.WorkerVolumeEncryption.KMSAccountID)
```

Comment:
if the `workerPool.WorkerVolumeEncryption.KMSAccountID` is an empty string, should we still use it with the `d.Set`?